.github/workflows/analysis_workflow.yml

```diff
@@ -57,12 +67,13 @@ jobs:
                 -DOPENEXR_RUN_FUZZ_TESTS='OFF' \
                 -DOPENEXR_ENABLE_THREADING='ON'
         working-directory: _build
-      - name: Build
+      - name: Build OpenEXR with build-wrapper
         shell: bash
         run: |
-          cmake --build . \
-                --target install \
-                --config Release
+          export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${PWD}/Half:${PWD}/Imath
+          echo ${PWD}
+          echo ${LD_LIBRARY_PATH}
+          build-wrapper-linux-x86-64 --out-dir bw_output make clean all
```

Comment:
Where is build-wrapper-linux-x86-64? Is there a reason to prefer invoking it rather than invoking cmake directly?

Reply:
Found the answer: the build wrapper is a SonarCloud tool that is required for the sonar scan:
https://docs.sonarqube.org/latest/analysis/languages/cfamily/
See prerequisites section:
"Analysis of C/C++/Objective-C projects requires the Build Wrapper. It runs the build and gathers all the configuration required for correct analysis of C/C++/Objective-C projects (such as macro definitions, include directories, …). The Build Wrapper does not impact your build; it merely eavesdrops on it and writes what it learns into files in a directory you specify."

Reply:
Got it, good to know.  To be more consistent, maybe we should replace the "make clean all" with "cmake --build . --target install --config Release", since that's how the other CI steps invoke the build.

Reply:
Noted, will change!